{"name": "test_greater", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert 100 > 100", "trace": "@pytest.mark.xfail\n    @pytest.mark.great\n    def test_greater():\n        num = 100\n>       assert num > 100\nE       assert 100 > 100\n\ntest_compare.py:8: AssertionError"}, "start": 1678268172304, "stop": 1678268172304, "uuid": "d8823a4b-4d36-42c4-8d55-806e53b80915", "historyId": "ae2daacf9362564e93898689c64e3912", "testCaseId": "9f2619453c551a5ac409fc05ab4afa43", "fullName": "test_compare#test_greater", "labels": [{"name": "tag", "value": "great"}, {"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "suite", "value": "test_compare"}, {"name": "host", "value": "BAN-LAP-VREDDY"}, {"name": "thread", "value": "7936-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_compare"}]}